bigtable/admin.go

```diff
@@ -667,60 +667,89 @@ func (ac *AdminClient) TableInfo(ctx context.Context, table string) (*TableInfo,
 	return ti, nil
 }
 
-type gcPolicySettings struct {
+type updateFamilyOption struct {
 	ignoreWarnings bool
 }
 
-// GCPolicyOption is the interface to change GC policy settings
-type GCPolicyOption interface {
```

Comment:
Ideally, we try to avoid already exported and released types. This would break customer's code

https://pkg.go.dev/cloud.google.com/go/bigtable#GCPolicyOption

But I would like to know what any of the admins think

@codyoss @quartzmo

Reply:
I agree. I would suggest just updating the documentation.

Reply:
Done.